opm/parser/eclipse/EclipseState/Schedule/Well.hpp

```diff
@@ -86,6 +86,15 @@ namespace Opm {
         WellPolymerProperties          getPolymerPropertiesCopy(size_t timeStep) const;
         const WellPolymerProperties&   getPolymerProperties(size_t timeStep) const;
 
+        bool getRFT(size_t time_step) const;
```

Comment:
I guess these should be called something like `getRFTActive() ` - because what you get/set is a flag as to whether a simulated RFT should be stored - not the actual RFT.